pubspec.yaml

```diff
@@ -24,6 +24,10 @@ dependencies:
   flutter:
     sdk: flutter
   overlay_screen: ^1.2.0+1
+  flutter_markdown: ^0.5.2
+  path_provider: ^1.6.27
+  url_launcher: ^5.7.10
+  shared_preferences: 0.5.12+4
```

Comment:
I'm not familiar with each of these packages? Can you explain what these do?

Reply:
* `path_provider` is to access images in your devices using absolute path. These images are not in flutter asset. I used this to test out the markdown file but now I don't think it's necessary. I'll delete it right away. However, later on, this might be useful when we implement the import image function in this app.
* `url_launcher` : when user clicks a link on app, this should open the device browser with the url. This package is to do this
* `shared_preferences` : to share features among app from time to time. Like when user wants to turn on dark mode, it should remember this so they don't have to change the dark mode setting every time they use the app.

Reply:
Cool! Thanks for letting me know about this! I think each of these packages are super important, but we should definitely relay with our team about `shared_preferences`, because it sounds like some persistent storage mechanism that we should run through everyone else!